tests/queries/0_stateless/03221_create_if_not_exists_setting.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
pls rework it as a sql test. to assert server errors you may use special syntax like in `tests/queries/0_stateless/02966_topk_counts_approx_count_sum.sql` for example

Reply:
Just did, thanks for the help. I'm now guessing `.sql` tests run as a locked transaction, but `.sh` tests aren't? And that's why my `.sh` tests failed when run in parallel? Seems like the parallel tests were creating/deleting each other's databases/tables.

Reply:
Actually nevermind, I discovered the `no-parallel` flag that does it.